ssl/ssl_ciph.c

```diff
@@ -498,12 +499,12 @@ static void do_load_builtin_compressions(void)
         }
     }
     CRYPTO_mem_ctrl(CRYPTO_MEM_CHECK_ENABLE);
+    return 1;
```

Comment:
Please look at the whole function.  If `sk_SSL_COMP_new()` fails, `ssl_comp_methods` will be NULL, *and that's already handled*